src/Illuminate/Http/Resources/Json/JsonResource.php

```diff
@@ -231,4 +231,24 @@ public function jsonSerialize()
     {
         return $this->resolve(Container::getInstance()->make('request'));
     }
+
+    /**
+     * Get a subset of the resource's attributes.
+     *
+     * @param array|mixed $attributes
```

Comment:
this type doc isn't correct. array is already a subtype of mixed. just write `mixed`.